.travis.yml

```diff
@@ -22,7 +22,7 @@ addons:
       - percona-toolkit
 
 install:
-  - go get -u github.com/Masterminds/glide
+  - curl https://glide.sh/get | sh
```

Comment:
`master` branch contains development version, it can make our build not reproducible.
(Well, IMO, `vendor/` should be committed to git, but that's another story.)